user-data

```diff
@@ -6,10 +6,16 @@ stylus:
   site:
     paletteEndpoint: ""
     registrationURL: ""
+    edgeHostToken: ""
 stages:
   boot:
     - hostname: localhost
-  after-install:
+  before-install:
+    - name: "Remove machine-id"
+      if: '[ -f "/run/cos/live_mode" ]'
+      commands:
+      - echo -n > /etc/machine-id
+  after-install.after:
```

Comment:
kairos docs still say `after-install`, is `after-install.after` actually a thing in stylus now?

Reply:
This is a Kairos thing. We are now ustlising the after-install step for some install hand-off, so our check in Packer needs to be moved to after-install.after so all the hand off logic completes before we power off